i18n/ja_JP/LC_MESSAGES/Alltube.po

```diff
@@ -53,11 +53,11 @@ msgstr "よく使われる形式"
 
 #: templates/info.tpl:35
 msgid "Best"
-msgstr "最高品質"
+msgstr "最高品質(mp4形式)"
 
 #: templates/info.tpl:36
 msgid "Remux best video with best audio"
-msgstr "最高品質に再エンコード"
+msgstr "最高品質の音声と動画をmkvに再エンコード"
```

Comment:
MKV is not mentioned in the source string.
If you think the format should be mentioned, please open an issue about this, but here is not the place to add it.